release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1061,5 +1068,165 @@ module openconfig-evpn {
+        enum UP {
+          description "VXLAN EVPN Overlay peer state is UP";
+        }
+        enum DOWN {
+          description "VXLAN EVPN Overlay peer state is DOWN";
+        }
+      }
+      description "VXLAN EVPN Overlay peer state";
+    }
+    leaf uptime {
+      type yang:timestamp;
+      description "VXLAN EVPN Overlay Peer Uptime";
+    }
+    leaf control-plane-vnis {
+      type string;
+      description "VNIs learned over the control-plane";
+    }
+    leaf router-mac {
+      type oc-yang:mac-address;
+      description "Peer Router mac-address";
+    }
   }
+
+  grouping evpn-endpoint-vni-state {
+    description "Grouping for VXLAN EVPN Overlay endpoint to vni mapping data";
+    leaf vni {
+      type oc-evpn-types:evi-id;
+      description "VNI Identifier";
+    }
+    leaf multidestination-traffic {
+      type union {
+        type oc-evpn-types:multicast-address;
+        type enumeration {
+          enum STATIC_INGRESS_REPLICATION {
+            description
+              "Static ingress replication mode.";
+          }
+          enum BGP_INGRESS_REPLICATION {
+            description
+              "BGP EVPN ingress replication mode. It includes the ability to
+              signal a P2MP LSP for the EVPN Inclusive Provider Tunnel
+              for BUM traffic";
+          }
+        }
+      }
+      description
+        "The data plane for overlays needs to handle the transport of
+         multidestination traffic. Multidestination traffic is typically
+         referred to as “BUM,” which stands for broadcast, unknown unicast,
+         or multicast. The two most common methods that can accommodate this
+         replication and transport in the underlay are IP multicast and
+         ingress replication
+         (also called head-end replication or unicast mode).";
+    }
+    leaf learning-mode {
+      type oc-evpn-types:learning-mode;
+      description "Learning mode";
+    }
+    leaf vni-type {
+      type enumeration {
+        enum L2 {
+          description "L2 VNI Type";
+        }
+        enum L3 {
+          description "L3 VNI Type";
+        }
+      }
+      description "VNI Type";
+    }
+    leaf vni-state {
+        type enumeration {
+          enum UP {
+            description "VNI state is UP";
+          }
+          enum DOWN {
+            description "VNI state is DOWN";
+          }
+        }
+        description "L2/3 VNI State";
+      }
+    leaf svi-state {
+        type enumeration {
+          enum UP {
+            description "SVI state is UP";
+          }
+          enum DOWN {
+            description "SVI state is DOWN";
+          }
+        }
+        description "SVI State used for VLAN to L3 VNI mapping";
+      }
+    leaf bridge-domain {
+        type uint32;
+        description "VLAN or BD that maps to this L2 VNI";
+      }
+
+    leaf l3-vrf-name {
+        type string;
+        description "VRF routing instance that maps to this L3 VNI";
```

Comment:
Same question as above here - how does this map in the context of OpenConfig?

Reply:
This leaf is not referencing another network instance but rather it reflects the VRF routing instance that has been configured to map to this particular Layer3 VNI that is used for routing between subnets in a VXLAN fabric.

Reply:
Updated description